CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 - Support for only simulating one baseline per redundant group
+- Checking the beam kernel width makes sense for Airy beams
```

Comment:
Can you rebase and check that this line ends up in the right place? I'm a little worried it'll be wrong after the recent version release.